public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -102,9 +332,23 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
                 {...register('groupInterval', evaluateEveryValidationOptions)}
               />
             </Field>
+
             {checkEvaluationIntervalGlobalLimit(watch('groupInterval')).exceedsLimit && (
               <EvaluationIntervalLimitExceeded />
             )}
+            {rulerRuleRequests && (
+              <>
+                <div>List of rules that belong to this group</div>
+                <div className={styles.evalRequiredLabel}>
+                  #Evaluations column represents the number of evaluations needed before alert starts firing.
+                </div>
```

Comment:
alert or alert rule? I also don't think we need needed..so "the number of evaluations before an alert rule starts firing."

Reply:
I think it's the alert what starts firing, not the alert rule. WDT @konrad147 ?

Reply:
this is what we have as a definition: When an alert rule is actively firing, that’s an alert